module1/projects/index.md

```diff
@@ -6,10 +6,12 @@ title: Module 1 - Projects
 ## Required Projects
 The following projects will be assigned and their completion is required to be promoted to module 2.
 
-<!-- Comment the following projects back in as they are checked and deemed ready to go! -->
-<!-- - Week 1 (Ungraded): [Credit Check](./credit_check.markdown) -->
-<!-- - Week 1-2 (Solo): [Flash Cards](./flashcards/) -->
-<!-- - Week 2-3 (Solo): [DMV](./dmv/) -->
-<!-- - Week 3-4 (Paired): [Battleship](./battleship/) -->
-<!-- - Week 5-6 (Group Final): [Futbol](./futbol/) -->
+
+- Week 1 (Ungraded): [Credit Check](./credit_check.markdown)
+<!-- Alternate between Flash Cards and War or Peace for repeaters -->
+- Week 1-2 (Solo): [Flash Cards](./flashcards/)
+- Week 2-3 (Solo): [DMV](./dmv/)
+<!-- Option to add more advanced option with Connect Four as other pair project -->
```

Comment:
Do we want to add a link here for War or Peace and Connect Four but leave them commented out so they're easy to set up the following inning?

Reply:
Oh great call. I can do that and push!